### ISSUE TITLE
sosa or sosa-ext for SOSA/SSN Extensions?

### ISSUE THREAD

**Issue by kjano** [w3c/sdw#1245]
I am unsure whether it would not be better to change something like sosa:hasMember into a sosa-ext namespace to avoid confusion, also because they make use of different modeling practices such as RDF vs schema.org domains and ranges. I already got several questions about this. IMHO, your (wonderful) extensions should be in their own namespace and import SOSA. For instance, I got asked why sosa:hasMember is not in the ontology, and I had to explain that it is in another ontology altogether.

**Comment by dr-shorthair** [comment]
When ssn-ext was published as a W3C Draft we decided to put the new terms in the existing SOSA namespace - see explanation here: https://www.w3.org/TR/vocab-ssn-ext/#namespaces

This is because the intention is for these terms to get rolled into the main vocabulary, in due course, so better for them to be used with their intended URIs than change them later or have a vocabulary composed of multiple namespaces, though I acknowledge that SSN already does this. I also acknowledge that almost any choice will be confusing to someone (and there is misleading statement in Paragraph 3 of the Status section, which betrays the fact that there have been various opinions about this and confusion remains!). 

Namespaces are a tricky thing. I used to mint new namespaces frequently, but my feeling now is that the W3C vocabularies have been a bit too precious about namespaces, and the result is a confusing proliferation which has not gone down well in the marketplace. Compare with Schema.org which is a single namespace, and OBO in which the main ontologies are in a single namespace (though OBO fakes it with a prefix on the local name). This is not a deal-breaker for me, however, as I think there is a downside on every choice.

**Comment by dr-shorthair** [comment]
Note that there are precedents for later recommendations adding new terms into existing W3C namespaces, including `rdf:` itself. 

For example [the `rdf:JSON` datatype](https://www.w3.org/TR/json-ld11/#the-rdf-json-datatype) and [the `rdf:CompoundLiteral` class and `rdf:language` and `rdf:direction` properties](https://www.w3.org/TR/json-ld11/#the-rdf-compoundliteral-class-and-the-rdf-language-and-rdf-direction-properties) are added in [JSON-LD 1.1 (2020)](https://www.w3.org/TR/json-ld11).

**Comment by maximelefrancois86** [comment]
We have slash-based IRIs for `sosa:`. I's therefore technically feasible to redirect some new SOSA term to a separate ontology (say https://w3.org/ns/sosa/ext` , that imports SOSA and extend it with other terms and axioms.

This was one of the motivations to push for one unique namespace for SOSA SSN and SSN-System, but the group voted against this approach. Now if I understand well, with ssn-ext you would want to use this approach ?

**Comment by dr-shorthair** [comment]
Yes - too many namespaces is counter-productive. See schema.org for counter-example.
